src/Apps/Collect/Components/Collection/Header/__tests__/DefaultHeader.test.tsx

```diff
@@ -70,7 +70,7 @@ describe("default header component", () => {
   it("renders images in the header component ", () => {
     const wrapper = getWrapper(props)
 
-    expect(wrapper.find("Image").length).toBe(5)
+    expect(wrapper.find("Image").length).toBe(10)
```

Comment:
Same as above - it's not obvious to me why this test would have gone from requiring 5 images to fill the header to requiring 10. Can you help me understand? I would have expected this not to change, since we're only placing as many images as it takes to fill the header.

Reply:
I'm returning more works than are in the set to handle window resizing.

Reply:
What will the effect be for a user on mobile who is viewing a collection with many header images? Will their device download only the number of images they can see, or will it download all (or 10) header images, only 3 of which they can see?

Also - I'm going to merge this as-is, since it fixes a crash. We can follow up with my question elsewhere.

Reply:
Good question, the tests indicate that for a mobile device a user might have up to 4 images to process, and depending on wide their screen is they might see only 3, or 3 and part of 4 since the overflow is acceptable and the banner is intended to look like its a row of repeating images. I have a ticket in the backlog to trigger a component update on window re-size, since the hook isn't working as expected. But this should be merged pretty soon so we don't have collections pages timing out.